configs/yolov5/yolov5_s-v61-attn0011-drop0011_syncbn_8xb16-300e_coco.py

```diff
@@ -0,0 +1,251 @@
+_base_ = '../_base_/default_runtime.py'
```

Comment:
It's better to inherit the `_base_ = ./yolov5_s-v61_syncbn_8xb16-300e_coco.py` config?